example/operator/20-garden.yaml

```
@@ -216,6 +216,7 @@ spec:
     #   featureGates:
     #     SomeGardenerFeature: true
     #   goAwayChance: 0.001
+    #   adminKubeconfigMaxExpiration: 1000h
```

Can you put the default value in this example manifest?
```suggestion
    #   adminKubeconfigMaxExpiration: 24h
```